src/main/kotlin/org/rust/ide/actions/RsFmtFileAction.kt

```diff
@@ -48,7 +48,16 @@ class RsFmtFileAction : DumbAwareAction() {
         try {
             cargo.reformatFile(module, file)
         } catch (e: ExecutionException) {
-            project.showBalloon(e.message ?: "", NotificationType.ERROR)
+            // #1131 - Check if we get a `no such subcommand: fmt` and let the user know to install fmt
+            val messageCheck = Regex("no such subcommand: fmt");
```

Comment:
In Kotlin, there's a handy `toRegex` extension, so it's more idiomatic to write `"no such subcommand: fmt".toRegex()`.

However in this case the regex is trivial, so we can just search for a substring, using the `in` operator:

```
if ("no such subcommand: fmt" in e.message) {
    ...
}
```